Client/Frontend/Theme/LegacyThemeManager/ThemedWidgets.swift

```diff
@@ -56,7 +56,7 @@ class ThemedTableViewCell: UITableViewCell, ThemeApplicable {
         // Take view model color if it exists, otherwise fallback to default colors
         textLabel?.textColor = viewModel?.textColor ?? theme.colors.textPrimary
         detailTextLabel?.textColor = viewModel?.detailTextColor ?? theme.colors.textSecondary
-        backgroundColor = viewModel?.backgroundColor ?? theme.colors.layer5
+        backgroundColor = viewModel?.backgroundColor ?? theme.colors.layer2
```

Comment:
Hi @mleers! I had a look into the issue a little more. Turns out this would have impact elsewhere, so I'll change what I said earlier: 
- Let's fix the problem instead by adjusting the color inside `CustomSearchEngineTextView` with `textField.backgroundColor` to be `layer5` instead of `layer2`. So the problem will be fixed only in that cell's case and won't affect other settings colors.

Thanks for looking into this!

Reply:
Addressed